src/main/java/nextstep/subway/application/StationService.java

```diff
@@ -49,4 +49,9 @@ public List<StationResponse> findAllStations() {
     public void deleteStationById(Long id) {
         stationRepository.deleteById(id);
     }
+
+    public Station getStationById(Long stationId) {
```

Comment:
```suggestion
    @Transactional(readOnly = true)
    public Station getStationById(Long stationId) {
```

조회만 하는 경우 위와 같이 설정할 수 있겠네요 ㅎㅎ

Reply:
아래와 같이 service 최상단에 선언해두었습니다!
```java
@Service
@Transactional(readOnly = true)
public class StationService
```

Reply:
최상단에 아래왁 같이 선언해두어서 저장 또는 수정하는 곳에서만 트랜잭션 선언하도록 했습니다
```java
@Service
@Transactional(readOnly = true)
public class LineService {
```